pkg/release/binaries/tar.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package binaries
+
+// Tar is used for accessing and interacting with a .tar file containing
+// a binary file stored on disk.
+type Tar struct {
```

Comment:
This is not correct, it represents a single binary that's already been extracted to disk, no?

Reply:
I see.... i got confused here by tars in tar